examples/restrict-by-media-type/restrict_by_media_type.description

```diff
@@ -1 +1 @@
-// This sample demonstrates the behaviour of the `Consumes` and `Produces` annotation attributes of the `resourceConfig` in Ballerina.
+// This sample demonstrates the behaviour of the `consumes` and `produces` annotation attributes of the `ResourceConfig` in Ballerina.
```

Comment:
Shall we rewrite this description? Something like below?
"Resources of HTTP services can be configured to restrict the types of media it consumes and produces. This is done through the `consumes` and `produces` annotation attributes of the `ResourceConfig` annotation used with resources."

@chamil321 thoughts? Also, I think we should include what happens when it receives an unsupported media type

Reply:
We have added a HTTP request in the client.sh to depict the negative scenario of consumes/produces